### PR TITLE
Create Mozgvya.com.xml

### PR DESCRIPTION
**Pull request by ryanbr**
Domain is used for bat-ads @ Brave.

### CODE REVIEW

**Comment by pipboy96** [comment]
Please use tabs instead of three spaces.

**Comment by ryanbr** [comment]
> Please use tabs instead of three spaces.

Some pull requests are using (single) spaces, and tab spacing can also vary on editor/platform. What is the general consensus?

**Comment by pipboy96** [comment]
@ryanbr For now, I guess it's either one tab or four spaces. Two spaces is sometimes used. Three spaces, if any other ruleset file has it, is extremely rare, I have never encountered it.

**Comment by ryanbr** [comment]
I hope that's better, tab's adjusted to 4 columns.

File: src/chrome/content/rules/Mozgvya.com.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<ruleset name="mozgvya.com">
+	<target host="mozgvya.com" />
+	<target host="www.mozgvya.com" />
```
Comment (pipboy96) [inline src/chrome/content/rules/Mozgvya.com.xml]: `dig www.mozgvya.com` = `NXDOMAIN`.

**Comment by pipboy96** [comment]
Good otherwise.

File: src/chrome/content/rules/Mozgvya.com.xml (hunk 1 above)
Comment (ryanbr) [reply src/chrome/content/rules/Mozgvya.com.xml]: okay

File: src/chrome/content/rules/Mozgvya.com.xml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+	Non-active domain:
+	- www.mozgvya.com
+-->
+<ruleset name="mozgvya.com">
+	<target host="mozgvya.com" />
+
+	<securecookie host=".+" name=".+" />
+
+	<rule from="^http:" to="https:" />
+<ruleset>
```
Comment (pipboy96) [inline src/chrome/content/rules/Mozgvya.com.xml]: `</ruleset>`

Comment (pipboy96) [inline src/chrome/content/rules/Mozgvya.com.xml]: `</ruleset>`

**Pull request merged**